Jenkinsfile

```diff
@@ -59,6 +61,7 @@ pipeline {
         }
```

Comment:
If you set the clone of infr_apps to use tag v2.0.0 on line 38 and then get the library checks passing, that should get everything into the correct state to move to the new jenkins function seamlessly.